UnityProject/Assets/Scripts/Items/Implants/Organs/Ears.cs

```diff
@@ -95,6 +95,8 @@ private void CheckPressure()
 				UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, CheckPressure);
 			}
 
+			if (RelatedPart.HealthMaster.playerScript.OrNull()?.playerMove.IsVisible == false) return;
```

Comment:
wouldn't it be better to get the gas container there they are in if in a locker or the the gas tile The Locker is on

Reply:
This assumes that it's just going to be containers that will cause this issue.
This solution covers all problems when players are sent to 0,0